### PR TITLE
Book - Simplify adding a team member

### PR DESCRIPTION
**Pull request by jomey**
An observed challenge over the years was to add many team members to the website at once without creating the dreaded merge conflict. This breaks up the monolithic `team.yaml` file and has each team member fill out their own. This hopefully makes it easier and has the nice side-effect of reusable files across hackweeks and years.

### CODE REVIEW

File: team/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# The team folder
+
+All members of the organizing team should add themselves to the webpage and
+this folder contains a file for each person describing themselves.
+
+## Adding a new member
+Follow these steps to add yourself:
+* Use the provided 'template.yaml' file and copy the file.
+* Rename the file to your name
```
Comment (aaarendt) [inline team/README.md:9]: Suggest deleting line 8 and replacing 9 - 12 with:
* Create a copy of the 'template.yaml' file and rename the file to your name:
```
 cp template.yaml FirstName_LastName.yaml
```

File: team/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+   ```
+    mv template.yaml FirstName_LastName.yaml
+   ```
+* Open the file in your favorite editor and fill out the details,
```
Comment (aaarendt) [inline team/README.md:13]: For clarity, specify which file to edit (so people don't start editing the template)
* Open the new file you just created....

File: team/template.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  organizations:
+    - name: Your institution
+  bio: A short bio about yourself.
+    You can have multiple lines. Just watch that you have two leading spaces at the beginning.
+  expertise:
+    - skill 1
+    - skill 2
+  social:
+    - icon: github
+      icon_pack: fab
```
Comment (aaarendt) [inline team/template.yaml:13]: Any way we can tell people to leave this as 'fab'? It's not immediately obvious to a new person.

**Review by aaarendt — APPROVED** [review]
This is awesome! I tested it locally and was able to add a new person seamlessly.

File: team/template.yaml (hunk 3 above)
Comment (jomey) [reply team/template.yaml:13]: We could add an inline comment

**Pull request merged**